src/upm_stdvector.i

```diff
@@ -0,0 +1,61 @@
+%include "autobox.i"
```

Comment:
Yes we should rename this to either upm_javavector.i / upm_javastdvector.i / upm_stdjavavector.i
Which sounds more appropriate?